sdk/resourcemanager/neonpostgres/armneonpostgres/neonpostgres_live_test.go

```diff
@@ -0,0 +1,191 @@
+	testsuite.subscriptionId = recording.GetEnvVariable("AZURE_SUBSCRIPTION_ID", "00000000-0000-0000-0000-000000000000")
+	testsuite.organizationName = "testogname"
+	resourceGroup, _, err := testutil.CreateResourceGroup(testsuite.ctx, testsuite.subscriptionId, testsuite.cred, testsuite.options, testsuite.location)
+	testsuite.Require().NoError(err)
+	testsuite.resourceGroupName = *resourceGroup.Name
+	fmt.Println("testsuite.resourceGroupName:", testsuite.resourceGroupName)
+	testsuite.Prepare()
+}
+
+func (testsuite *NeonpostgresTestSuite) TearDownSuite() {
+	testsuite.CleanUp()
+	_, err := testutil.DeleteResourceGroup(testsuite.ctx, testsuite.subscriptionId, testsuite.cred, testsuite.options, testsuite.resourceGroupName)
+	testsuite.Require().NoError(err)
+	time.Sleep(time.Second * 3)
+	testutil.StopRecording(testsuite.T())
+}
+
+func TestNeonpostgresTestSuite(t *testing.T) {
+	suite.Run(t, new(NeonpostgresTestSuite))
+}
+
+func (testsuite *NeonpostgresTestSuite) TestOptionList() {
+	cred, err := azidentity.NewDefaultAzureCredential(nil)
+	if err != nil {
+		log.Fatalf("failed to obtain a credential: %v", err)
+	}
+	ctx := context.Background()
+	clientFactory, err := armneonpostgres.NewClientFactory(testsuite.subscriptionId, cred, testsuite.options)
+	if err != nil {
+		log.Fatalf("failed to create client: %v", err)
+	}
+	pager := clientFactory.NewOperationsClient().NewListPager(nil)
+	for pager.More() {
+		page, err := pager.NextPage(ctx)
+		if err != nil {
+			log.Fatalf("failed to advance page: %v", err)
+		}
+		for _, v := range page.Value {
+			fmt.Println("page value", v)
+		}
+		page = armneonpostgres.OperationsClientListResponse{
+			OperationListResult: armneonpostgres.OperationListResult{
+				Value: []*armneonpostgres.Operation{
+					{
+						Name:         to.Ptr("ekvuojrpbuyogrikfuzyghio"),
+						IsDataAction: to.Ptr(true),
+						Display: &armneonpostgres.OperationDisplay{
+							Provider:    to.Ptr("lvppuskqcdcoejwuvsqrloczvnouy"),
+							Resource:    to.Ptr("rvvd"),
+							Operation:   to.Ptr("odjjqnodcgszczpsdrhrpwmqssrybr"),
+							Description: to.Ptr("bwmstukaiaoisiu"),
+						},
+						Origin:     to.Ptr(armneonpostgres.OriginUser),
+						ActionType: to.Ptr(armneonpostgres.ActionTypeInternal),
+					},
+				},
+				NextLink: to.Ptr("https://contoso.com/nextlink"),
+			},
+		}
+	}
+}
+
+func (testsuite *NeonpostgresTestSuite) Prepare() {
+	cred, err := azidentity.NewDefaultAzureCredential(nil)
+	if err != nil {
+		log.Fatalf("failed to obtain a credential: %v", err)
+	}
+	clientFactory, err := armneonpostgres.NewClientFactory(testsuite.subscriptionId, cred, testsuite.options)
+	if err != nil {
+		log.Fatalf("failed to create client: %v", err)
+	}
+
+	_, err = clientFactory.NewOrganizationsClient().BeginCreateOrUpdate(testsuite.ctx, testsuite.resourceGroupName, testsuite.organizationName, armneonpostgres.OrganizationResource{
```

Comment:
i think it should not be placed in prepare. it is a test case for create or update.